### PR TITLE
fixed Unable to add >50% of available product qty to order in admin p…

### PR DESCRIPTION
**Pull request by keyurk-emipro**
### Description (*)

Fixed Unable to add >50% of available product qty to order in admin panel

### Fixed Issues (if relevant)

1. magento/magento2#18245: Unable to add >50% of available product qty to order in admin panel
2. ...

### Manual testing scenarios (*)

1. In the admin panel, select `Sales > Orders > Create New Order`
2. Select the sample "Veronica Costello" customer.
3. Select "Add Products".
4. Add any product with 50% + 1 Qty (Ex. Product Qty = 100 then add 51 Qty in Order)
5. Now you will not get warning message and Product will be added successfully.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @keyuremipro. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by aleron75 — APPROVED** [review]

File: app/code/Magento/CatalogInventory/Model/Quote/Item/QuantityValidator/QuoteItemQtyList.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function getQty($productId, $quoteItemId, $quoteId, $itemQty)
             $this->_checkedQuoteItems[$quoteId][$productId]['items']
         )
         ) {
-            $qty += $this->_checkedQuoteItems[$quoteId][$productId]['qty'];
+            $qty = $this->_checkedQuoteItems[$quoteId][$productId]['qty'];
```
Comment (sivaschenko) [inline app/code/Magento/CatalogInventory/Model/Quote/Item/QuantityValidator/QuoteItemQtyList.php:39]: I think the goal of this operation is to accumulate all product quantities allocated for all quote items, so in my opinion `+=` is exactly what needs to be used here.

Possibly the cause of the issues is incorrect `quoteItemId` used for some of the calls of this method?

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @keyuremipro thanks for the pull request. Please see my code review comment

**Comment by sidolov** [comment]
@keyuremipro , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**